demos/serverless/README.md

```diff
@@ -28,6 +28,21 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings Namespace vs Chime Namespace
+The AWS.Chime and the AWS.ChimeSDKMeetings are both Amazon Chime's AWS clients to help builders create the Amazon Chime SDK meeting, create attendees, and enable optional meeting features such as transcription. AWS.ChimeSDKMeetings is intended to replace the previous AWS.Chime client.
+
+Introduced a new command line parameter `-u` or `--use-chime-sdk-meetings` which is a boolean flag and can used to specify the AWS client. This flag is true by default to make AWS.ChimeSDKMeetings as default client. Also, this flag is primarly intended for internal testing of the legacy Chime SDK client.
```

Comment:
Suggest to re-phrase this to:
Use `-u` or `--use-chime-sdk-meetings` boolean to specify the AWS client. It defaults to `true` using the `AWS.ChimeSDKMeetings` client. This flag is primarily intended for internal testing of the legacy `AWS.Chime` client.

Reply:
updated